lib/gruf/autoloaders.rb

```diff
@@ -24,12 +24,12 @@ class << self
       include Enumerable
 
       ##
-      # Initialize the autoloaders with a given controllers path
+      # Initialize the autoloaders with a given controllers paths
       #
-      # @param [String] controllers_path The path to Gruf Controllers
+      # @param [Array<String>] controllers_paths The path to Gruf Controllers
       #
-      def load!(controllers_path:)
-        controllers(controllers_path: controllers_path)
+      def load!(controllers_paths:)
```

Comment:
Hi @salmanasiddiqui - while this is a good change to support multiple paths, this unfortunately makes a backwards incompatible change here I'd like to avoid.

Can we add this as a _new_ keyword argument, and deprecate the old one (to be removed in gruf 3.0). We should support both syntaxes in 2.x.